integration-tests/hibernate-reactive-mariadb/pom.xml

```diff
@@ -150,109 +150,6 @@
-                        <groupId>io.fabric8</groupId>
-                        <artifactId>docker-maven-plugin</artifactId>
-                        <configuration>
-                            <images>
-                                <image>
-                                    <name>healthcheck-${mariadb.image}</name>
-                                    <alias>quarkus-test-mariadb</alias>
-                                    <build>
-                                        <from>${mariadb.image}</from>
-                                        <healthCheck>
-                                            <!-- The exact values for these aren't very important, but it is important they are there -->
-                                            <interval>5s</interval>
-                                            <timeout>3s</timeout>
-                                            <startPeriod>5s</startPeriod>
-                                            <retries>5</retries>
-                                            <!--  We could also use /usr/local/bin/healthcheck.sh but it seemed complicated to get the right level.
-                                            Note that mysqladmin ping returns 0 even if the password is wrong so we don't need to pass in a password, but it makes the logs cleaner-->
-                                            <cmd>
-                                                <shell>mysqladmin ping -h localhost -u root -psecret|| exit 1</shell>
-                                            </cmd>
-                                        </healthCheck>
-                                    </build>
-                                    <run>
-                                        <network>
-                                            <mode>bridge</mode>
-                                        </network>
-                                        <ports>
-                                            <port>3308:3306</port>
-                                        </ports>
-                                        <env>
-                                            <MYSQL_USER>hibernate_orm_test</MYSQL_USER>
-                                            <MYSQL_PASSWORD>hibernate_orm_test</MYSQL_PASSWORD>
-                                            <MYSQL_DATABASE>hibernate_orm_test</MYSQL_DATABASE>
-                                            <MYSQL_RANDOM_ROOT_PASSWORD>true</MYSQL_RANDOM_ROOT_PASSWORD>
-                                        </env>
-                                        <log>
-                                            <prefix>MariaDB:</prefix>
-                                            <date>default</date>
-                                            <color>cyan</color>
-                                        </log>
-                                        <!-- Speed things up a bit by not actually flushing writes to disk -->
-                                        <tmpfs>/var/lib/mysql</tmpfs>
```

Comment:
Note we're losing this particular optimization, which I suppose might make CI runs slightly longer.

I think we could reintroduce it by calling `setTmpFsMapping()` somewhere around here:

https://github.com/quarkusio/quarkus/blob/74bbe72eb06cbf5aaf47d0f342cb25ab9a08f8b7/extensions/devservices/mariadb/src/main/java/io/quarkus/devservices/mariadb/deployment/MariaDBDevServicesProcessor.java#L111

But I'm not sure we want that for all our users?

Reply:
I've raised https://github.com/quarkusio/quarkus/issues/51126 to track